server/app/views/v1/grid_stats/stats.json.jbuilder

```diff
@@ -0,0 +1,23 @@
+json.stats @stats.each do |stat|
+  json.container_id stat[:container].name
+  if !stat[:stats].nil?
+    json.cpu do
+      json.usage stat[:stats].cpu['usage_pct']
+      json.limit stat[:stats].spec['cpu']['limit']
+      json.num_cores ContainerStat.calculate_num_cores(stat[:stats].spec['cpu']['mask'])
+    end
+    json.memory do
+      json.usage stat[:stats].memory['usage']
+      json.limit stat[:stats].spec['memory']['limit']
+    end
+    json.network do
+      json.name stat[:stats].network['name']
+      json.rx_bytes stat[:stats].network['rx_bytes']
+      json.tx_bytes stat[:stats].network['tx_bytes']
+    end
+  else
+    json.cpu nil
```

Comment:
@nevalla : The CLI actually expects this to be null/missing when calling the `service stats` command.  In the latest version of this PR I am sharing the same view for all stats APIs so I would either have to change this in the CLI or leave it alone.